pytest_tap/plugin.py

```diff
@@ -20,7 +20,7 @@
 
 def pytest_addoption(parser):
     """Include all the command line options."""
-    group = parser.getgroup('terminal reporting', after='general')
+    group = parser.getgroup('terminal reporting', 'reporting', after='general')
```

Comment:
That change was accidentally checked in. It actually does nothing. It would have changed a section header in the output to pytest --help from "terminal reporting" to "reporting" if an identical call didn't already exist in the pytest code.